base_tier_validation/models/res_users.py

```diff
@@ -15,7 +15,6 @@ def review_user_count(self):
         domain = [
             ("status", "=", "pending"),
             ("can_review", "=", True),
-            ("id", "in", self.env.user.review_ids.ids),
```

Comment:
this

Reply:
Had to remove this, to make the tests pass.
I'm not seeing in the code where the values to review_ids are written. 
(expect for the no longer needed lines in the test)
`self.assertTrue(self.test_user_1.get_reviews({"res_ids": review.ids}))`
With no valid ID assigned it broke the tests..

(Filtering on the user ID is done, a couple llines below.)

Reply:
Maybe some more refactoring is needed on the domain. (Or an alternative function to write `review_ids`) I'm not sure..

Reply:
This needs to stay, otherwise the review count will include all users' reviews I fear. The test can be fixed with
```
--- a/base_tier_validation/tests/test_tier_validation.py
+++ b/base_tier_validation/tests/test_tier_validation.py
@@ -417,7 +417,7 @@ class TierTierValidation(CommonTierValidation):
         # Request validation
         review = test_record.with_user(self.test_user_2).request_validation()
         self.assertTrue(review)
-        test_record.invalidate_model()
+        self.env.invalidate_all()
         self.assertTrue(test_record.review_ids)
         # Used by front-end
         count = self.test_user_1.with_user(self.test_user_1).review_user_count()
```

Reply:
Thanks, that fixed it!